pg-promise-exercises/exercises.js

```diff
@@ -0,0 +1,224 @@
+const pg = require('pg-promise')()
+const assert = require('assert')
+
+const postgresConfig = {
+  host: 'localhost',
+  port: 5432,
+  database: 'pg-promise-exercises',
+  user: '<change-this-to-your-username>', // replace this with your username
+  password: '' //  replace this if you have set a password for your username (this is unlikely)
+};
+
+
+const db = pg(postgresConfig);
+
+/* -----------------------------------------
+   Exercise 1
+   -----------------------------------------
+
+   This is an example function that finds all the books from the `books` table
+   @function: `allBooks`
+   @input params: None
+   @output: [{id, title, author_id, subject_id}]
+
+   The assertion fails, and you have to make it pass.
+
+*/
+
+
+
+const allBooks = db.any('select * from books')
+/* This is calling the `then` function on the `allBooks` promise, and checks if
+   we get back 15 rows. This assertion will fail. Make it PASS!*/
+allBooks.then(books => {
+  assert.deepEqual(books.length, 20)
+}).catch(error => {
+  console.log('Dang, my assertion failed.', error);
+});
+
+/* --------End of Exercise 1---------------- */
+
+
+
+
+
+/* -----------------------------------------
+           Exercise 2
+   -----------------------------------------
+
+   Implement the function `firstTenBooks` which returns just the names of the
+   books, and make the assertion pass.
+   @function: `firstTenBooks`
+   @input params: None
+   @output: [{id, title, author_id, subject_id}]
+
+
+*/
+
+let firstTenBooks; // = .... IMPLEMENT THIS FUNCTION
+firstTenBooks.then(books => {
+  assert(books.length, 10)
```

Comment:
this function doesn't seem to be testing for just the titles of the first ten books returning.